runtime/doc/lsp.txt

```diff
@@ -753,6 +753,8 @@ set_log_level({level})                               *vim.lsp.set_log_level()*
                 Levels by name: "TRACE", "DEBUG", "INFO", "WARN", "ERROR"
                 Level numbers begin with "TRACE" at 0
 
+                Set to 99 or "OFF" to disable logging
```

Comment:
can we just add it to `log.levels` as level 6? There will never be a level higher than ERROR.